src/pages/index.js

```diff
@@ -322,3 +329,23 @@ deleteForm.addEventListener("submit", handleDeleteSubmit);
 editFormElement.addEventListener("submit", handleEditFormSubmit);
 cardForm.addEventListener("submit", handleAddCardSubmit);
 avatarForm.addEventListener("submit", handleAvatarSubmit);
+
+document.addEventListener("click", (evt) => {
+  if (evt.target === document.querySelector(".modal_opened")) {
```

Comment:
Please, don’t add listeners to `document` or `window` if you don’t remove them because the listeners always watch any clicks on the page and make checks. That’s a waste of resources.

* And the `overlay` handler was correct in your project 6. Please, have a look at your code there https://snipboard.io/dJAVWa.jpg

Just come your own code from project 6. You don't need to recreate new features